.github/workflows/mako.yml

```diff
@@ -0,0 +1,37 @@
+name: mako-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+  mako-testsuite-1_1_4:
+    runs-on: ubuntu-latest
+    env:
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: mako
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: sqlalchemy/mako
+          ref: rel_1_1_4
+          path: mako
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install tox
+        run: pip install tox
+      - name: Create tox env
+        run: tox -e py --notest
+      - name: Inject ddtrace
+        run: .tox/py/bin/pip install ../ddtrace
+      - name: Add pytest configuration for ddtrace
+        run: echo -e "[pytest]\nddtrace-patch-all = 1" > pytest.ini
+      - name: Run tests
+        run: tox -e py
+        
```

Comment:
no idea why this is still showing missing newline at the end of file. might need to tweak your editor.

Reply:
Is there a formatter or linter I should set up?

Reply:
I'm just using vscode